crates/dotrain/src/parser/rainlangdocument/mod.rs

```diff
@@ -198,7 +198,7 @@ mod tests {
     fn test_process_operand_method() -> anyhow::Result<()> {
         let mut rl = RainlangDocument::new();
         let namespace = HashMap::new();
-        let exp = "<12 56>";
+        let exp = r#"<"12." 56>"#;
```

Comment:
rather than overload an existing test, why not write several related tests and run them all?